runtime/runtime-async/src/commonMain/kotlin/app/cash/sqldelight/async/logs/LogAsyncSqlDriver.kt

```diff
@@ -0,0 +1,110 @@
+package app.cash.sqldelight.async.logs
+
+import app.cash.sqldelight.async.AsyncQuery
+import app.cash.sqldelight.async.AsyncTransacter
+import app.cash.sqldelight.async.db.AsyncSqlCursor
+import app.cash.sqldelight.async.db.AsyncSqlDriver
+import app.cash.sqldelight.async.db.AsyncSqlPreparedStatement
+
+class LogAsyncSqlDriver(
```

Comment:
lets just remove this from the async runtime, its not so valuable that I think we should start with it, and maybe theres a better event driven abstraction or something we can replace the normal runtime one with

Reply:
The async runtime tests use it so I'll just move it to the test sources, otherwise we'd have zero tests.